phoenix-core/src/it/java/org/apache/phoenix/end2end/DateTimeIT.java

```diff
@@ -18,20 +18,7 @@
 package org.apache.phoenix.end2end;
 
 import static org.apache.phoenix.query.QueryConstants.MILLIS_IN_DAY;
-import static org.apache.phoenix.util.TestUtil.ATABLE_NAME;
```

Comment:
The import preferences don't look correct. For guidelines see the code conventions section here: http://phoenix.apache.org/contributing.html